components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -190,26 +199,6 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
           // Sort naturally first by name, then by id
           this.users = ChefSorters.naturalSort(users, ['name', 'id']);
         });
-
-    // If, however, the user browses directly to /settings/teams/ID, the store
-    // will not contain the team data, so we fetch it.
-    // TODO: This also fires when teamFromRoute fires; should inhibit that since
-    // we already have the team details, as noted above.
-    // Triggered every time the route changes (including on initial load).
-    combineLatest([
-      this.store.select(routeParams),
-      this.store.select(routeURL)
-    ]).pipe(
-      takeUntil(this.isDestroyed),
-      map(([params, url]) => [params.id as string, url]),
-      // Only fetch if we are on the team details route, otherwise
-      // we'll trigger GetTeam with the wrong input on any route
-      // away to a page that also uses the :id param.
-      filter(([id, url]) => TEAM_DETAILS_ROUTE.test(url) && id !== undefined),
-      // Remove the url because we only need to check if the id has changed
-      map(([id, _url]) => id),
-      distinctUntilChanged()
-    ).subscribe(id => this.store.dispatch(new GetTeam({ id })));
  }
```

Comment:
(nit) I think it helps readability within `ngOnInit` to have a blank line after each block (a couple of them already do).